Expensify/App
# Translate alternate text to Spanish

### Details

[Attachment] alternate text in LHN is not translated to Spanish

### Fixed Issues

$ https://github.com/Expensify/App/issues/7440

### Tests


- Send attachment
- Change language to Spanish and check at LHN. [Attachment] alternate text should be translated.

### QA Steps


- Send attachment
- Change language to Spanish and check at LHN. [Attachment] alternate text should be translated.

### Tested On


### Screenshots


#### Web

https://user-images.githubusercontent.com/25876548/152380204-014b0631-9e43-4b26-8e9b-c3fb46a97ebe.mp4

#### Mobile Web

![Screenshot_20220203-212457_Chrome](https://user-images.githubusercontent.com/25876548/152380678-4cff0b29-102e-4fac-9cac-9a5527e61e8b.jpg)
![Screenshot_20220203-212428_Chrome](https://user-images.githubusercontent.com/25876548/152380736-39000e16-96bf-41ad-8b01-a18b2de3f4c7.jpg)

#### Desktop

<img width="1440" alt="Screen Shot 2022-02-09 at 21 34 22" src="https://user-images.githubusercontent.com/25876548/153238711-63015198-67ea-4d8d-82fa-8190ce265a0c.png">
<img width="1440" alt="Screen Shot 2022-02-09 at 21 34 35" src="https://user-images.githubusercontent.com/25876548/153238834-f54f533a-e9d9-42bb-8193-8f421c6bac83.png">


#### iOS

![Simulator Screen Shot - iPhone 13 - 2022-02-09 at 21 37 45](https://user-images.githubusercontent.com/25876548/153239068-30ea2056-b59b-4f50-8a19-d6a174f6b821.png)
![Simulator Screen Shot - iPhone 13 - 2022-02-09 at 21 37 27](https://user-images.githubusercontent.com/25876548/153239302-9baa981f-8cd4-46e4-bddd-d732eeb11175.png)


#### Android

![Screenshot_20220203-200522_New Expensify](https://user-images.githubusercontent.com/25876548/152380776-5ecb0336-7b32-4ad6-9e88-31264d76e5d8.jpg)
![Screenshot_20220203-200602_New Expensify](https://user-images.githubusercontent.com/25876548/152380791-c136c27c-2c30-4e62-8730-b9f27cd5adb9.jpg)
